RULES/core/flags.go

```diff
@@ -3,22 +3,19 @@ package core
 import (
 	"encoding/json"
 	"fmt"
-	"hash/crc32"
 	"io/ioutil"
 	"os"
-	"sort"
+	"path"
+	"path/filepath"
 	"strconv"
-	"strings"
 )
 
-const configFilePath = "../FLAGS.json"
+const flagsFileName = "FLAGS.json"
```

Comment:
As a general comment I don't think it's a good idea to mix configuration file formats. `MODULE` is a YAML file and, therefore, `FLAGS` should also have been a YAML file. That would have made it much easier for the user to migrate flags from one to the other without having to mess with the syntax.